src/components/molecules/PlayaTime/PlayaTime.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { useState } from "react";
+import dayjs from "dayjs";
+import timezone from "dayjs/plugin/timezone";
+import updateLocale from "dayjs/plugin/updateLocale";
+import utc from "dayjs/plugin/utc";
+
+import { CURRENT_TIME_IN_LOCATION, CURRENT_TIMEZONE } from "settings";
+
+import { useInterval } from "hooks/useInterval";
+
+import "./PlayaTime.scss";
+
+dayjs.extend(utc);
+dayjs.extend(timezone);
+dayjs.extend(updateLocale);
```

Comment:
Already resonating sentiment https://doesitsparkle.slack.com/archives/C01ER4M61TL/p1629705277150800 
I would, however, argue this change might be late and OOS for the BM, so it will be better to scrutinize it while backporting to staging/master.

Reply:
Going with the @debt suggestion https://github.com/sparkletown/sparkle/pull/2010/files/96ed8ae6689f7ee335db626431b731002e77aa16#r695395989

Reply:
yep, makes sense. 🙃

Reply:
done in 0700a1d